third_party/xla_client/pjrt_computation_client_test.cc

```diff
@@ -86,4 +86,14 @@ TEST(PjRtComputationClientTest, Init) {
       result_literals[0]));
 }
 
+TEST(PjRtComputationClientTest, DeviceOrdinals) {
+  // Get CPU devices
+  tsl::setenv("PJRT_DEVICE", "CPU", true);
+  tsl::setenv("CPU_NUM_DEVICES", 2, true);
+  auto client = std::make_unique<PjRtComputationClient>();
+
+  // Verify that device global ordinals are dense starting at 0.
+  ASSERT_EQ(client->GetAllDevices(), {"CPU:0", "CPU:1"});
```

Comment:
Without this change, what's the results?

Reply:
It would be the same, CPUs already have dense ordinals without the remapping. This is a poor unit test, but we're constrained on what we can test without mocking the wrapped PjRtClient. I'm looking into this, but we may take adding a mock here as a follow-up item.